backend/api/elite2Api.js

```diff
@@ -15,17 +15,14 @@ const elite2ApiFactory = client => {
 
   const put = (context, url, data) => client.put(context, url, data).then(processResponse(context))
 
-  const userLocations = context => get(context, 'api/users/me/locations')
-  const currentUser = context => get(context, 'api/users/me')
-  const userRoles = context => get(context, 'api/users/me/roles')
-  const userCaseLoads = context => get(context, 'api/users/me/caseLoads')
+  const userLocations = context => (context.authSource !== 'auth' ? get(context, 'api/users/me/locations') : [])
```

Comment:
would have preferred this to be only if `authSource === 'nomis'`, however that would require all three apps being released at once.  Can do it this way and then change later on